.github/templates/dagger-docker.yml

```diff
@@ -0,0 +1,68 @@
+name: Dagger/Docker Testing Suite
+
+on:
+  schedule:
+    # Every night at 4:00 AM UTC
+    - cron: '0 4 * * *'
+  workflow_dispatch:
+    inputs:
+      BASE_IMAGE_TAG:
+        description: 'Image tag to use for the Docker image'
+        required: true
+      PAVICS_HOST:
+        description: 'Pavics URL to test against'
+        required: true
+        default: 'https://pavics.ouranos.ca/'
+      SANITIZE_FILE_URL:
+        description: 'URL for the sanitizer configuration'
+        required: true
+        default: 'https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/raw/master/notebooks/output-sanitize.cfg'
+
+concurrency:
+  # For a given workflow, if we push to the same branch, cancel all previous builds on that branch except on main.
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+env:
+  BASE_IMAGE_TAG: 230601
```

Comment:
This is now `py311-250423-update250730` at this moment.  But it will change again !

Maybe we should have a file in the PAVICS-e2e-workflow-tests repo with this value and use the value from that file, on `master` branch?